src/librustc_error_codes/error_codes/E0703.md

```diff
@@ -0,0 +1,18 @@
+Invalid ABI(Application Binary Interface) used in the code.
+
+Erroneous code example:
+
+```compile_fail,E0703
+extern "invalid" fn foo() {} //~ ERROR
```

Comment:
No need for the \~ (for info, it's used in the UI tests, no need here). You can just write instead:

`extern "invalid" fn foo() {} // error!`